tests/conftest.py

```diff
@@ -106,3 +106,5 @@ def pytest_collection_modifyitems(items):
             item.add_marker(pytest.mark.datasets)
         if "/transformers/" in path:
             item.add_marker(pytest.mark.transformers)
+        if "/horovod/" in path:
+            item.add_marker(pytest.mark.horovod)
```

Comment:
What does this pytest marker do? (just curious)

Reply:
It marks all the tests inside `/horovod` directory with the `horovod` marker so we can selectively run only the tests inside this directory with `horovodrun ... pytest -m horovod`.